wasm/src/program/transaction.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+  account::{Address, PrivateKey},
+  record::RecordPlaintext
+};
+use crate::{
+    Aleo,
+    Identifier,
+    Process,
+    Program,
+    ProvingKey,
+    TransactionNative
+};
+
+use std::str::FromStr;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct TransactionBuilder {}
+
+#[wasm_bindgen]
+impl TransactionBuilder {
+    /// Creates an execute transaction from a full proof of execution
+    pub fn build_transfer_full(
+        private_key: PrivateKey,
+        proving_key: ProvingKey,
+        address: Address,
+        amount: u64,
+        record: RecordPlaintext,
+    ) -> String {
```

Comment:
This should be a result enum so a more informative error can be piped up to js.

```suggestion
    ) -> Result<String, String> {
```